modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/ProfileFragment.kt

```diff
@@ -76,6 +82,8 @@ class ProfileFragment : BaseFragment() {
     @Inject lateinit var analyticsTracker: AnalyticsTracker
 
     private val viewModel: ProfileViewModel by viewModels()
+    private val referralsViewModel: ReferralsViewModel by viewModels()
+    private var referralsCountBadge: BadgeDrawable? = null
```

Comment:
Is it needed to keep a reference to this object? Can't it be kept as a local `val` inside of the `onViewCreated()`?

Reply:
Converted to local in 2c8a273.